CONTRIBUTING

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+Hey, thanks for your interest in contributing to the NYU CS Wiki! Please review this document **before making a pull request**.
+
+## Guidelines and Style
+
+- Follow a second ('you can...') or third ('it is..') person point of view when writing articles
+- Make sure to cite your sources; this can be done [via markdown](https://www.markdownguide.org/extended-syntax/#footnotes). For an example, look at the ``Holodeck: High Speed Research Network`` article citations.
+- Keep it (mostly) professional
+
+## 👷 Development 
+
+You will need node/npm installed and then need to run ``npm install`` in the project directory (this will grab all the dependencies). From here, you can create your own articles/modifications to the source in either the /docs or /src directories, respectively. After, to run the project: 
+
+- 🚧 Development (does not include search index):
```

Comment:
Maybe add this will hot reload so any changes you make in the code will be directly updated on the development website (and is generally preferred over building unless you need the search index).